AUTHORS

```diff
@@ -0,0 +1,935 @@
+Guido Zuidhof <me@guido.io>
+HAMIDx9 <me@hamidx9.ir>
+Habib Rehman <HR@users.noreply.github.com>
+Hailin Tseng <arniu@users.noreply.github.com>
+Hajime Morrita <omo@dodgson.org>
+Hanae Kae <riel.kaestera@gmail.com>
+Hank Bao <hankbao84@gmail.com>
+Hans Kristian Flaatten <hans@starefossen.com>
+Hansen Zhang <hansen.s.zhang@gmail.com>
+Haojian Wu <hokein.wu@gmail.com>
+Hari Juturu <jhreddy@microsoft.com>
+Hari Juturu <juturu@live.com>
+Hari Krishna Reddy Juturu <jhreddy@microsoft.com>
+HariJ <harij@HariJMAC.guest.corp.microsoft.com>
+HariJ <harij@HariJMAC.local>
+Harry <1174941773@qq.com>
+Hashed Hyphen <hashedhyphen@gmail.com>
+HayateLaTech <broladin@gmail.com>
+Heejin Lee <appetizermonster@users.noreply.github.com>
+Heilig Benedek <benecene@gmail.com>
+Hein Haraldson Berg <hein@haraldsonberg.net>
+Hiroshige Umino <hiroshige88@gmail.com>
+Hiroshige Umino yaotti <yaotti@qiita.com>
+Howard.Zuo <leftstick@qq.com>
+Hugh Kennedy <hughskennedy@gmail.com>
+Hugo Mano <hugo@hugomano.com>
+Hum4n01d <hum4n01d@icloud.com>
+Ian Langworth ☠ <ian@artillery.com>
+Ian Olsen <iolsen@github.com>
+Ian Ornelas <ian.orn@gmail.com>
+IgorKlopov <igor@klopov.com>
+Importcjj <importcjj@gmail.com>
+Indrek Ardel <indrek@ardel.eu>
+Ingo Richter <ingo.richter@gmail.com>
+Ionică Bizău <bizauionica@gmail.com>
+Ivan Kulikov <gunmetal313@gmail.com>
+Ivan Mir <ivmirx@fastmail.fm>
+Ivan Mir <ivmirx@users.noreply.github.com>
+Ivan Žužak <ivan.zuzak@github.com>
+Ivo Georgiev <ivo@strem.io>
+Ivo von Putzer Reibegg <ivo.putzer@gmail.com>
+J.M <jm@jublo.net>
+JP Richardson <jprichardson@gmail.com>
+Jacob Groundwater <groundwater@github.com>
+Jacob Groundwater <groundwater@gmail.com>
+Jacob Page <jacob.h.page@gmail.com>
+Jake <JakeH@users.noreply.github.com>
+Jakukyo Friel <weakish@gmail.com>
+James Ide <ide@jameside.com>
+James Kyburz <james.kyburz@gmail.com>
+James Wheare <james@wheare.org>
+Jan Němeček <enterthenamehere@gmail.com>
+Jan Wiemers <j.wiemers@allerdesign.de>
+Jason Poon <git@jasonpoon.ca>
+Jason Zhekov <jasssonpet@gmail.com>
+Javan Makhmali <javan@javan.us>
+Jeff Rehbein <jeff.rehbein@code42.com>
+Jeff Wear <jeff@mixmax.com>
+Jeffrey Morgan <jmorganca@gmail.com>
+Jeffrey Morgan <usabilityetc@gmail.com>
+Jeremy Apthorp <nornagon@nornagon.net>
+Jeroen Visser <jeroenvisser101@users.noreply.github.com>
+Jesse Grosjean <jesse@hogbaysoftware.com>
+Jessica Lord <jessica.lord@github.com>
+Jessica Lord <jlord@github.com>
+Jesus David Rojas <jesus_19david@hotmail.com>
+Jhen <developer@jhen.me>
+Jiaji Zhou <jiaji.zh@gmail.com>
+Jimi (Dimitris) Charalampidis <jimikar@gmail.com>
+Jimmy Moon <ragingwind@gmail.com>
+JinHyeok Yeon <duswlsgur@daum.net>
+Joao Moreno <mail@joaomoreno.com>
+Joe Naha <joe.naha@gmail.com>
+Johan Giraldo <johanpalomino59@gmail.com>
+Johannes Schmitz <j13z@users.noreply.github.com>
+John Eismeier <32205350+jeis2497052@users.noreply.github.com>
+John Haley <john@haley.io>
+John Kleinschmidt <jkleinsc@github.com>
+John Kleinschmidt <kleinschmidtorama@gmail.com>
```

Comment:
@jkleinsc which email address would you prefer as your canonical one?

Reply:
@ckerr the GitHub one

Reply:
@jkleinsc Done